services/web/project/service/metrics_compilation_service.py

```diff
@@ -0,0 +1,67 @@
+from ..repository import metrics_repository, github_user_repository
+
+def compile(login):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'compile'](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5653140)